app/subscriber/src/features/search-page/SearchPage.tsx

```diff
@@ -150,27 +167,26 @@ export const SearchPage: React.FC<ISearchType> = ({ showAdvanced }) => {
             dateOffset: filter.dateOffset + 7,
           };
         }
-        const offSet = filter.dateOffset ? filter.dateOffset : 0;
-        setDateVisible(!(offSet > 1));
-        const dayInMs = 24 * 60 * 60 * 1000; // Hours*Minutes*Seconds*Milliseconds
-        let offSetDate = new Date();
-        offSetDate.setDate(offSetDate.getDate() - offSet);
-        offSetDate.setHours(0, 0, 0);
+        const offset = filter.dateOffset ?? 0;
```

Comment:
Fix the logic with offset and date values.  Now the results should be correctly displayed.